lib/csa/ccm/cli/command.rb

```diff
@@ -41,7 +41,7 @@ def xlsx2yaml(input_xlsx_file)
             return
           end
 
-          version = input_xlsx_file[/(?<=v)[0-9\.]*(?=-)/] || 'unknown'
+          version = Matrix.version_from_filepath(input_xlsx_file)
           matrix = Matrix.from_xlsx(version, input_xlsx_file)
```

Comment:
@CAMOBAP795 I don't think you need to `version` anymore because it can already run `Matrix.parse_version`. So `Matrix.from_xlsx(input_xlsx_file)` should be sufficient.